tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -8,154 +8,238 @@
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_AMER_INDIAN', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_ASIAN', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_BLACK', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_HAWAIIAN', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_WHITE', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='MARITAL_STATUS', result_function=validators.isInLimits(1, 5),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='PARENT_WITH_MINOR_CHILD', result_function=validators.isInLimits(1, 3),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='EDUCATION_LEVEL', result_function=validators.or_validators(
+                            validators.isInStringRange(0, 16),
+                            validators.isInStringRange(98, 99)
+                        ),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                        result_field='CITIZENSHIP_STATUS', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='COOPERATION_CHILD_SUPPORT', result_function=validators.oneOf((1, 2, 9)),
+                    ),
+                validators.validate__FAM_AFF__HOH__Fed_Time(),
```

Comment:
the logic here appears to be missing?
![2369p2](https://github.com/raft-tech/TANF-app/assets/63075587/e78b8a3e-6f41-491a-9329-b6e89caf0131)